apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/internal/ResponseReader.kt

```diff
@@ -21,7 +20,6 @@ interface ResponseReader {
 
   fun <T : Any> readObject(field: ResponseField, objectReader: ObjectReader<T>): T?
 
-  @JvmSynthetic
```

Comment:
Not sure I follow. Why removing that/Why did we need that in the first place ?

Reply:
This somehow allowed us not to implement these methods in Java since they were not visible to Jvm. I think using JvmDefault while targeting 1.8 would work but that's not available in Kotlin common.